src/rangefinder.jl

```diff
@@ -39,4 +39,27 @@ function rangefinder!(Y, A, Ω; q, Z, orthogonalize)
     end
 
     return nothing
-end
\ No newline at end of file
+end
+
+function chebyshev_rangefinder(A::AbstractMatrix{T}, ν::Float64, r::Int; q::Int=1, Ω=nothing, orthogonalize=true) where {T <: Number}
+    m, n = size(A)
+
+    isnothing(Ω) && (Ω = GaussianTestMatrix(m, r))
+
+    Y0 = qr(Ω.Ω).Q
```

Comment:
Does this step make the Chebyshev rangefinder incompatible with `SSFTTestMatrix`?

Reply:
Use Matrix(Ω) instead of Ω.Ω